src/Tribe/Views/V2/Views/Day_View.php

```diff
@@ -146,4 +146,60 @@ protected function build_url_for_date( $url_date, $canonical, array $passthru_va
 
 		return $url;
 	}
+
+	/**
+	 * {@inheritDoc}
+	 */
+	protected function setup_template_vars() {
+
+		$template_vars = parent::setup_template_vars();
+		$sorted_events = $this->sort_events( $template_vars['events'] );
+
+		$template_vars['events'] = $sorted_events;
+
+		return $template_vars;
+	}
+
+	/**
+	 * Add timeslot and sort events for the day view.
+	 *
+	 * Iterate over the day events to add timeslots and sort them.
+	 *
+	 * @since TBD
+	 *
+	 * @param array $events  An array of events.
+	 *
+	 * @return array The sorted and modified array.
+	 */
+	protected function sort_events( $events ) {
+
+		foreach ( $events as $event ) {
+
+			if ( ! empty( $event->all_day ) ) {
+				$event->timeslot = esc_html__( 'All Day', 'the-events-calendar' );
+			} else if( ! empty( $event->multiday ) ) {
```

Comment:
* :warning: Usage of ELSE IF is discouraged; use ELSEIF instead
* :no_entry_sign: There must be a space between if and the first parenthesis '('
* :no_entry_sign: Space after opening control structure is required
* :no_entry_sign: No space before opening parenthesis is prohibited
* :no_entry_sign: Expected 1 space(s) after IF keyword; 0 found

<sub>hash: 5b4bbcbe9b2072b4e2270e1307ccd916</sub>